packages/sdk/test/lib/StoredValue.test.ts

```diff
@@ -0,0 +1,132 @@
+import { TypedJSON } from 'typedjson';
+import { StoredValue } from '../../src/lib/StoredValue';
+import { expect } from 'chai';
+
+describe('StoredValue', () => {
+  it('should parse Account stored value correctly', function () {
```

Comment:
This test doesn't test anything. There should be some asserts at the end of the test, right?

Reply:
Yes, you're right. Done.